src/components/Checkbox/__tests__/index.tsx

```diff
@@ -74,23 +74,25 @@ describe('Checkbox', () => {
       </>,
     ))
 
-  test('renders with click event', async () => {
+  test('renders with click event', () => {
     const node = renderWithTheme(
       <Checkbox onChange={() => {}} size={37} value="test">
         Checkbox Label
       </Checkbox>,
+      {},
```

Comment:
We need this empty object ?

Reply:
yes otherwise it complains

Reply:
It shouldn't, option in renderWithTheme is optional